specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/device.json

```diff
@@ -0,0 +1,1044 @@
+              "Location": {
+                "description": "The location of the resource.",
+                "type": "string"
+              },
+              "retry-after": {
+                "description": "The retry interval in seconds.",
+                "type": "integer",
+                "format": "int32"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/ProvisioningServiceErrorDetails"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "API to register the device": {
+            "$ref": "./examples/device/RuntimeRegistration/registerDevice.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CertificateIssuanceResponse": {
+      "type": "object",
+      "properties": {
+        "issuedCertificate": {
+          "description": "The end-entity certificate issued to a client.",
+          "type": "string",
+          "readOnly": true,
+          "example": "-----BEGIN CERTIFICATE-----MIIEsjCCApqgAwIBAgICEAEwDQYJKoZIhvcNAQELBQAwJjEkMCIGA1UEAwwbQ09OVE9TT19URVNUX0lOVEVSTUVESUFURV8xMB4XDTIyMDQyNjE3NTIzNVoXDTIyMDUyNjE3NTIzNVowLjEsMCoGA1UEAwwjY29udG9zby1yZWdpc3RyYXRpb24taWQuY29udG9zby5jb20wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDDUXRyR46cUts0KkBSrYHW9PNth2wcgzX5r5jxI2L1RHgFmcLLFiJEA52ilz/2ahfjo5+uzNa7xO66DUVc5ukPrfu/uXxBZc4sXMzJLK3+yNBQrvaSBixWFg10W6r5hmzBDbDkJRtq75DVjTfUhJRhYbG2PadKj/GKA4JLIijKBo8XJHjcEwesdLF2tWxrNt8QvcUEtT4Ge3SehoTH/AHSQLen2/qAxzKYNKt2sdJgTFZ5uJOUGDdOOF7GstVHcKsnHS1CgUW7EwsH2XLfZ67j35bpOOJOocD4Mg1CRS7FDRcH9pBZ+Cng0aH0mGflijWCB0HQ/zHk4zt9e3eg7UcTAgMBAAGjgeEwgd4wCQYDVR0TBAIwADARBglghkgBhvhCAQEEBAMCBkAwMwYJYIZIAYb4QgENBCYWJE9wZW5TU0wgR2VuZXJhdGVkIFNlcnZlciBDZXJ0aWZpY2F0ZTAdBgNVHQ4EFgQUALTpWEUQKgoM5nkVZgMxNoeB3HMwRQYDVR0jBD4wPIAUqV8aE5IM1sTp8YUMaBXxKxUDYK+hIKQeMBwxGjAYBgNVBAMMEUNPTlRPU09fVEVTVF9ST09UggIQADAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDQYJKoZIhvcNAQELBQADggIBADOl1OSzOt7/I2GzY+NqM46ZcyDGWQsDlHq4I+fn6Jv0wX/x2wuMRjrh0WeT3gtUFD/MV0lqsk2O4YGvDavL3/XgwRbDBT9R0JaC86aCBBt6TUmq1g+lgx8Hh/ObllpZpYqbLIlFzWnuiFJonOv7fJ22y7Eta03wUZIJWqofXv5W1KKwscAxLrw0p6NWvwuRRKKtVp/wcJcBnjEh0uulHJYrgOiminPVmVfyWqht3VqZIesrHMrru5ITOVGpqLeSNpJnMBH3EQ6ClBCcPL6EDurIC/UgCUDMWjZKM2L4tNoaHwtFNo2KEBz7/4xloWix7+HzZdUL7Ly3WpFPk7UqOplKBKBni8KMbFMv8PgsKCu3/D8KTf1rHeEDhtRtOZrbi6yqxovD1YvLb8CqA6cNYn5+BiFKRGXCeXoI8l0qPGy4lXMRcUqdcYPD2gu98+vpa9zdB9OmLdhd7F3A2Y2yHULncaHKSFEnPmbE+sh+bg263gFXdoToVuvB5LeH1VPwaqINllCR6RyylKlKTF5Yy7N6NhvGbn5xxnFKvz7flgwdX1br4juJ430KeUCOzgkpyKoE0+WGQ91Vg5hkML27Ui90aLqQtlCClJwHalnOyQwvh8ZbGUdp0BJUbDhv0Bh5IF8Pkc7LLdMJUv/fEz3NDZyVJTpi23gdp/ebJhg5QbGE-----END CERTIFICATE-----"
```

Comment:
For brevity, since you already have an `x-ms-example` I'd elide these swagger-standard `example` properties.

Reply:
I recently discovered https://stoplight.io/open-source/prism and found that adding these examples has the effect of temporally decouples service and SDK teams. Essentially SDK teams can prepare their code and mock their service and be ready to test when the service is ready. I know I don't have the examples everywhere part of it are the core changes we need to make on our end so that the swagger files generate better with embedded examples. If you have an alternative to prism can use the examples as prepared by hand, I am all ears.

Reply:
Prism is an awesome tool and would love to hear more about how you have leveraged it.  As for examples, I think they have value (e.g. for documentation) even when there are x-ms-examples.

Reply:
Awesome, I will share my thoughts in a follow up review session. There is a nice process that we can build here to unleash productivity.